### PR TITLE
refactor: rename data gas to blob gas for relevant deneb fields

### PR DESCRIPTION
**Pull request by g11tech**
pursuant to 
 - https://github.com/ethereum/consensus-specs/pull/3461

renaming data gas fields => blob gas

TODO: 
- [x] spec tests update

### CODE REVIEW

**Review by dapplion — DISMISSED** [review]

File: packages/beacon-node/test/spec/specTestVersioning.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {DownloadTestsOptions} from "@lodestar/spec-test-util";
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 export const ethereumConsensusSpecsTests: DownloadTestsOptions = {
-  specVersion: "v1.4.0-beta.0",
+  specVersion: "v1.4.0-beta.1",
```
Comment (matthewkeil) [inline packages/beacon-node/test/spec/specTestVersioning.ts:18]: Out of curiosity, why do we have this magic string in this file and in `packages/params/test/e2e/ensure-config-is-synced.test.ts`?

**Review by matthewkeil — APPROVED** [review]
Changes lgtm 🚀 

I noticed that we do not have a test for 
`tests/core/pyspec/eth2spec/test/helpers/sharding.py` from the spec.  Is that the only spec we still need to add to our suite?

Also the question about the magic string for specVersion.  Not sure if those would ever want to be different or not.

File: packages/beacon-node/test/spec/specTestVersioning.ts (hunk 1 above)
Comment (g11tech) [reply packages/beacon-node/test/spec/specTestVersioning.ts:18]: this corresponds to protocol spec test releases (which go hand in hand with spec releases) : https://github.com/ethereum/consensus-spec-tests/releases

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.10.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.10.0) :tada:

**Pull request merged**